share/models/creative.py

```diff
@@ -12,18 +12,22 @@
 class AbstractCreativeWork(ShareObject, metaclass=TypedShareObjectMeta):
     title = models.TextField(blank=True)
     description = models.TextField(blank=True)
+    identifiers = ShareManyToManyField('Identifier', through='WorkIdentifier')
 
     # Used to determine if something should be surfaced in ES or not
     # this may need to be renamed later
     is_deleted = models.BooleanField(default=False)
 
+    # True for works that only exist because they were referenced by other works.
+    # Should be false if there's any information beyond an identifier.
+    # Don't put stubs in ES.
+    is_stub = models.BooleanField(default=False)
```

Comment:
I'm don't think this field is necessary. We can just define a minimum set of fields required to be indexed into elastic if this does become a problem. I just foresee a lot of issues involving either flipping this by mistake or forgetting to set it to false.

I see why this would be nice to have in the database but it's a field that is dependant upon the state of other fields. Any case like that I'd prefer to use `@property` to define them and forfeit the ability to filter it at the database level., that way it's impossible to be out of sync.

Side note: While it's a super edge case, I could see the use case for finding mostly empty records in elasticsearch to fill them out based on where the identifier resolve to.